sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleDisableLocalDateConvertersIT.java

```diff
@@ -0,0 +1,89 @@
+package org.sdase.commons.server.morphia;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import dev.morphia.Datastore;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.sdase.commons.server.mongo.testing.MongoDbRule;
+import org.sdase.commons.server.morphia.test.Config;
+import org.sdase.commons.server.morphia.test.model.Person;
+import org.sdase.commons.server.testing.DropwizardRuleHelper;
+import org.sdase.commons.server.testing.LazyRule;
+
+/** Tests if entities can be added by exact definition. */
+public class MorphiaBundleDisableLocalDateConvertersIT {
+
+  private static final MongoDbRule MONGODB = MongoDbRule.builder().build();
+
+  private static final LazyRule<DropwizardAppRule<Config>> DW =
+      new LazyRule<>(
+          () ->
+              DropwizardRuleHelper.dropwizardTestAppFrom(MorphiaTestApp.class)
+                  .withConfigFrom(Config::new)
+                  .withRandomPorts()
+                  .withConfigurationModifier(
+                      c -> c.getMongo().setHosts(MONGODB.getHost()).setDatabase("testPeople"))
+                  .build());
+
+  @ClassRule public static final RuleChain CHAIN = RuleChain.outerRule(MONGODB).around(DW);
+
+  @Before
+  public void cleanCollection() {
+    getDatastore().delete(getDatastore().createQuery(Person.class));
```

Comment:
We have a method on MongoDB rule for this

Reply:
Thx. I didn't remember this. I will amend the change when rebasing.

Reply:
Done